Initial version of decrypt/export and Github action (#1)
ashb/gh-action-encrypted-secrets: README.rst

=================

> encrypted-secrets

Reviewer: ```suggestion
Encrypted secrets
```

Just cause it's a title

Author: It's also the module name though so... :shrug:

=================

> Inspired by hiera-eyaml, but for iteration in to Python projects

Reviewer: ```suggestion
Inspired by [hiera-eyaml](https://github.com/voxpupuli/hiera-eyaml), but for iteration into Python projects
```

Author: Yes, except this is RST :)
